README.md

```diff
@@ -1,6 +1,74 @@
 # eos-icons-react
 [![Open Source Love svg2](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badges/)
 
+## Using eos-icons-react in your projects
+Refer to [EOS Icons](eos-icons.com) for more details about all the icons that are supported by EOS-Icons React
+
+[Storybook EOS-Icons React](https://storybook.eos-icons.com/), checkout storybook for an in-depth look at the various different icons that are provided by EOS-Icons React as well the props available for configuring icons.
+
+### Installation
+```
+# yarn
+yarn add eos-icons-react
+# npm
+npm install eos-icons-react
+```
+
+### Usage
+#### common icons usage
+```
+import { EOS_STAR } from 'eos-icons-react';
+
+<EOS_STAR />
+```
+#### Filled/Outlined icons usage
+```
+import { EOS_STAR_FILED, EOS_STAR_OUTLINED } from 'eos-icons-react';
```

Comment:
Spelling of `FILLED` here and line 28

Reply:
Thanks @sharmaaditya570191 for pointing this out